Build error on master (57943c4)

I cannot build the current `master (57943c4)` on Arch Linux.
My configure is:
```
    ../configure --prefix=/usr \
                 --target=x86_64-w64-mingw32 \
                 --enable-lto \
                 --disable-nls \
                 --with-system-readline \
                 --with-expat \
                 --with-python=python3.8
```

Build Error:

```log
 CXX    p-exp.o
  CXX    pdb.o
../../gdb/pdb.c:77:24: error: ‘R_PDB’ was not declared in this scope; did you mean ‘R2_SDB’?
   77 | static std::unique_ptr<R_PDB>
      |                        ^~~~~
      |                        R2_SDB
../../gdb/pdb.c:77:29: error: template argument 1 is invalid
   77 | static std::unique_ptr<R_PDB>
      |                             ^
../../gdb/pdb.c:77:29: error: template argument 2 is invalid
../../gdb/pdb.c: In function ‘int get_r_pdb(const string&)’:
../../gdb/pdb.c:80:3: error: ‘R_PDB’ was not declared in this scope; did you mean ‘R2_SDB’?
   80 |   R_PDB pdb = {nullptr};
      |   ^~~~~
      |   R2_SDB
../../gdb/pdb.c:83:29: error: ‘pdb’ was not declared in this scope; did you mean ‘llvm::pdb’?
   83 |       if (init_pdb_parser (&pdb, path.c_str ()))
      |                             ^~~
      |                             llvm::pdb
In file included from ../../gdb/pdb.h:4,
                 from ../../gdb/pdb.c:16:
/usr/include/llvm/DebugInfo/PDB/Native/PDBFile.h:31:11: note: ‘llvm::pdb’ declared here
   31 | namespace pdb {
      |           ^~~
../../gdb/pdb.c:91:18: error: cannot convert ‘std::nullptr_t’ to ‘int’ in return
   91 |           return nullptr;
      |                  ^~~~~~~
../../gdb/pdb.c:109:20: error: cannot convert ‘std::nullptr_t’ to ‘int’ in return
  109 |             return nullptr;
      |                    ^~~~~~~
../../gdb/pdb.c:115:20: error: cannot convert ‘std::nullptr_t’ to ‘int’ in return
  115 |             return nullptr;
      |                    ^~~~~~~
../../gdb/pdb.c:136:24: error: cannot convert ‘std::nullptr_t’ to ‘int’ in return
  136 |                 return nullptr;
      |                        ^~~~~~~
../../gdb/pdb.c:149:38: error: ‘pdb’ was not declared in this scope; did you mean ‘llvm::pdb’?
  149 |       if (init_pdb_parser_with_buf (&pdb, r_buffer))
      |                                      ^~~
      |                                      llvm::pdb
In file included from ../../gdb/pdb.h:4,
                 from ../../gdb/pdb.c:16:
/usr/include/llvm/DebugInfo/PDB/Native/PDBFile.h:31:11: note: ‘llvm::pdb’ declared here
   31 | namespace pdb {
      |           ^~~
../../gdb/pdb.c:157:10: error: cannot convert ‘std::nullptr_t’ to ‘int’ in return
  157 |   return nullptr;
      |          ^~~~~~~
../../gdb/pdb.c: At global scope:
../../gdb/pdb.c:245:35: error: ‘R_PDB’ was not declared in this scope; did you mean ‘R2_SDB’?
  245 | static std::tuple<std::unique_ptr<R_PDB>, std::string>
      |                                   ^~~~~
      |                                   R2_SDB
../../gdb/pdb.c:245:40: error: template argument 1 is invalid
  245 | static std::tuple<std::unique_ptr<R_PDB>, std::string>
      |                                        ^
../../gdb/pdb.c:245:40: error: template argument 2 is invalid
../../gdb/pdb.c:245:54: error: template argument 1 is invalid
  245 | static std::tuple<std::unique_ptr<R_PDB>, std::string>
      |                                                      ^
../../gdb/pdb.c: In function ‘int load_pdb(objfile*)’:
../../gdb/pdb.c:250:54: error: cannot convert ‘<brace-enclosed initializer list>’ to ‘int’ in return
  250 |   if (paths.empty ()) return {nullptr, std::string ()};
      |                                                      ^
../../gdb/pdb.c:256:36: error: cannot convert ‘<brace-enclosed initializer list>’ to ‘int’ in return
  256 |         return {std::move (p), path};
      |                                    ^
../../gdb/pdb.c:259:34: error: cannot convert ‘<brace-enclosed initializer list>’ to ‘int’ in return
  259 |   return {nullptr, std::string ()};
      |                                  ^
../../gdb/pdb.c: In function ‘void read_pdb(objfile*, minimal_symbol_reader&)’:
../../gdb/pdb.c:291:19: error: ‘R_PDB’ was not declared in this scope; did you mean ‘R2_SDB’?
  291 |   std::unique_ptr<R_PDB> pdb;
      |                   ^~~~~
      |                   R2_SDB
../../gdb/pdb.c:291:24: error: template argument 1 is invalid
  291 |   std::unique_ptr<R_PDB> pdb;
      |                        ^
../../gdb/pdb.c:291:24: error: template argument 2 is invalid
../../gdb/pdb.c:293:47: error: no match for ‘operator=’ (operand types are ‘std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>’ and ‘int’)
  293 |   std::tie (pdb, pdb_path) = load_pdb (objfile);
      |                                               ^
In file included from /usr/include/c++/10.2.0/bits/unique_ptr.h:37,
                 from /usr/include/c++/10.2.0/memory:83,
                 from ../../gdb/gdbsupport/common-exceptions.h:25,
                 from ../../gdb/gdbsupport/common-defs.h:137,
                 from ../../gdb/defs.h:28,
                 from ../../gdb/pdb.h:1,
                 from ../../gdb/pdb.c:16:
/usr/include/c++/10.2.0/tuple:1173:7: note: candidate: ‘std::tuple<_T1, _T2>& std::tuple<_T1, _T2>::operator=(typename std::conditional<__assignable<const _T1&, const _T2&>(), const std::tuple<_T1, _T2>&, const std::__nonesuch&>::type) [with _T1 = int&; _T2 = std::__cxx11::basic_string<char>&; typename std::conditional<__assignable<const _T1&, const _T2&>(), const std::tuple<_T1, _T2>&, const std::__nonesuch&>::type = const std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&]’
 1173 |       operator=(typename conditional<__assignable<const _T1&, const _T2&>(),
      |       ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1175:35: note:   no known conversion for argument 1 from ‘int’ to ‘std::conditional<true, const std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&, const std::__nonesuch&>::type’ {aka ‘const std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&’}
 1173 |       operator=(typename conditional<__assignable<const _T1&, const _T2&>(),
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 1174 |          const tuple&,
      |          ~~~~~~~~~~~~~             
 1175 |          const __nonesuch&>::type __in)
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
/usr/include/c++/10.2.0/tuple:1184:7: note: candidate: ‘std::tuple<_T1, _T2>& std::tuple<_T1, _T2>::operator=(typename std::conditional<__assignable<_T1, _T2>(), std::tuple<_T1, _T2>&&, std::__nonesuch&&>::type) [with _T1 = int&; _T2 = std::__cxx11::basic_string<char>&; typename std::conditional<__assignable<_T1, _T2>(), std::tuple<_T1, _T2>&&, std::__nonesuch&&>::type = std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&&]’
 1184 |       operator=(typename conditional<__assignable<_T1, _T2>(),
      |       ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1186:30: note:   no known conversion for argument 1 from ‘int’ to ‘std::conditional<true, std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&&, std::__nonesuch&&>::type’ {aka ‘std::tuple<int&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&>&&’}
 1184 |       operator=(typename conditional<__assignable<_T1, _T2>(),
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 1185 |          tuple&&,
      |          ~~~~~~~~             
 1186 |          __nonesuch&&>::type __in)
      |          ~~~~~~~~~~~~~~~~~~~~^~~~
/usr/include/c++/10.2.0/tuple:1196:2: note: candidate: ‘template<class _U1, class _U2> std::__enable_if_t<__assignable<const _U1&, const _U2&>(), std::tuple<_T1, _T2>&> std::tuple<_T1, _T2>::operator=(const std::tuple<_U1, _U2>&) [with _U1 = _U1; _U2 = _U2; _T1 = int&; _T2 = std::__cxx11::basic_string<char>&]’
 1196 |  operator=(const tuple<_U1, _U2>& __in)
      |  ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1196:2: note:   template argument deduction/substitution failed:
../../gdb/pdb.c:293:47: note:   mismatched types ‘const std::tuple<_T1, _T2>’ and ‘int’
  293 |   std::tie (pdb, pdb_path) = load_pdb (objfile);
      |                                               ^
In file included from /usr/include/c++/10.2.0/bits/unique_ptr.h:37,
                 from /usr/include/c++/10.2.0/memory:83,
                 from ../../gdb/gdbsupport/common-exceptions.h:25,
                 from ../../gdb/gdbsupport/common-defs.h:137,
                 from ../../gdb/defs.h:28,
                 from ../../gdb/pdb.h:1,
                 from ../../gdb/pdb.c:16:
/usr/include/c++/10.2.0/tuple:1206:2: note: candidate: ‘template<class _U1, class _U2> std::__enable_if_t<__assignable<_U1, _U2>(), std::tuple<_T1, _T2>&> std::tuple<_T1, _T2>::operator=(std::tuple<_U1, _U2>&&) [with _U1 = _U1; _U2 = _U2; _T1 = int&; _T2 = std::__cxx11::basic_string<char>&]’
 1206 |  operator=(tuple<_U1, _U2>&& __in)
      |  ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1206:2: note:   template argument deduction/substitution failed:
../../gdb/pdb.c:293:47: note:   mismatched types ‘std::tuple<_T1, _T2>’ and ‘int’
  293 |   std::tie (pdb, pdb_path) = load_pdb (objfile);
      |                                               ^
In file included from /usr/include/c++/10.2.0/bits/unique_ptr.h:37,
                 from /usr/include/c++/10.2.0/memory:83,
                 from ../../gdb/gdbsupport/common-exceptions.h:25,
                 from ../../gdb/gdbsupport/common-defs.h:137,
                 from ../../gdb/defs.h:28,
                 from ../../gdb/pdb.h:1,
                 from ../../gdb/pdb.c:16:
/usr/include/c++/10.2.0/tuple:1216:2: note: candidate: ‘template<class _U1, class _U2> std::__enable_if_t<__assignable<const _U1&, const _U2&>(), std::tuple<_T1, _T2>&> std::tuple<_T1, _T2>::operator=(const std::pair<_U1, _U2>&) [with _U1 = _U1; _U2 = _U2; _T1 = int&; _T2 = std::__cxx11::basic_string<char>&]’
 1216 |  operator=(const pair<_U1, _U2>& __in)
      |  ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1216:2: note:   template argument deduction/substitution failed:
../../gdb/pdb.c:293:47: note:   mismatched types ‘const std::pair<_T1, _T2>’ and ‘int’
  293 |   std::tie (pdb, pdb_path) = load_pdb (objfile);
      |                                               ^
In file included from /usr/include/c++/10.2.0/bits/unique_ptr.h:37,
                 from /usr/include/c++/10.2.0/memory:83,
                 from ../../gdb/gdbsupport/common-exceptions.h:25,
                 from ../../gdb/gdbsupport/common-defs.h:137,
                 from ../../gdb/defs.h:28,
                 from ../../gdb/pdb.h:1,
                 from ../../gdb/pdb.c:16:
/usr/include/c++/10.2.0/tuple:1227:2: note: candidate: ‘template<class _U1, class _U2> std::__enable_if_t<__assignable<_U1, _U2>(), std::tuple<_T1, _T2>&> std::tuple<_T1, _T2>::operator=(std::pair<_U1, _U2>&&) [with _U1 = _U1; _U2 = _U2; _T1 = int&; _T2 = std::__cxx11::basic_string<char>&]’
 1227 |  operator=(pair<_U1, _U2>&& __in)
      |  ^~~~~~~~
/usr/include/c++/10.2.0/tuple:1227:2: note:   template argument deduction/substitution failed:
../../gdb/pdb.c:293:47: note:   mismatched types ‘std::pair<_T1, _T2>’ and ‘int’
  293 |   std::tie (pdb, pdb_path) = load_pdb (objfile);
      |                                               ^
../../gdb/pdb.c:298:10: error: base operand of ‘->’ is not a pointer
  298 |   if (pdb->pdb_parse (pdb.get ()))
      |          ^~
../../gdb/pdb.c:298:27: error: request for member ‘get’ in ‘pdb’, which is of non-class type ‘int’
  298 |   if (pdb->pdb_parse (pdb.get ()))
      |                           ^~~
../../gdb/pdb.c:308:14: error: base operand of ‘->’ is not a pointer
  308 |       l = pdb->pdb_streams2;
      |              ^~
../../gdb/pdb.c:373:6: error: base operand of ‘->’ is not a pointer
  373 |   pdb->finish_pdb_parse (pdb.get ());
      |      ^~
../../gdb/pdb.c:373:30: error: request for member ‘get’ in ‘pdb’, which is of non-class type ‘int’
  373 |   pdb->finish_pdb_parse (pdb.get ());
      |                              ^~~
```

--- a/.github/workflows/c-cpp.yml
+++ b/.github/workflows/c-cpp.yml
@@ -15,7 +15,7 @@ jobs:
         with:
           path: radare2
           repository: radareorg/radare2
-          ref: 4.5.0
+          ref: 5.0.0
 
       - name: Build radare2
         working-directory: radare2

--- a/gdb/pdb.c
+++ b/gdb/pdb.c
@@ -74,14 +74,14 @@ section_by_name (const char *name, struct objfile *objfile)
   return sect;
 }
 
-static std::unique_ptr<R_PDB>
+static std::unique_ptr<RPdb>
 get_r_pdb (const std::string & path)
 {
-  R_PDB pdb = {nullptr};
+  RPdb pdb = {nullptr};
   if (std::ifstream (path).good ())
     {
       if (init_pdb_parser (&pdb, path.c_str ()))
-        return std::make_unique<R_PDB> (pdb);
+        return std::make_unique<RPdb> (pdb);
     }
   else if (path.rfind ("target:", 0) == 0)
     {
@@ -149,7 +149,7 @@ get_r_pdb (const std::string & path)
       if (init_pdb_parser_with_buf (&pdb, r_buffer))
         {
           xfree (buffer);
-          return std::make_unique<R_PDB> (pdb);
+          return std::make_unique<RPdb> (pdb);
         }
 
       xfree (buffer);
@@ -242,7 +242,7 @@ get_pdb_paths (struct objfile *objfile)
   return paths;
 }
 
-static std::tuple<std::unique_ptr<R_PDB>, std::string>
+static std::tuple<std::unique_ptr<RPdb>, std::string>
 load_pdb (objfile *objfile)
 {
   auto paths = get_pdb_paths (objfile);
@@ -288,7 +288,7 @@ try_load_pdb_bfd (objfile *objfile)
 void
 read_pdb (struct objfile *objfile, minimal_symbol_reader & reader)
 {
-  std::unique_ptr<R_PDB> pdb;
+  std::unique_ptr<RPdb> pdb;
   std::string pdb_path;
   std::tie (pdb, pdb_path) = load_pdb (objfile);
 

